### PR TITLE
I18n implementation for application

### PR DESCRIPTION
**Pull request by euginios**
@ExFroggy @pshur @halfnext @RPE8

### CODE REVIEW

File: src/components/topTable/topTable.tsx [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-			<Button variant="contained">Button-1</Button>
+			<h1>{t("table.table1Text")}</h1>
+			<Button variant="contained">{t("buttonPattern", {number: 1})}</Button>
 			<Button variant="contained" disabled>
-				Button-2
+			{t("buttonPattern", {number: 2})}
 			</Button>
 			<Button variant="contained" color="success">
-				Button-3
+			{t("buttonPattern", {number: 3})}
```
Comment (ExFroggy) [inline src/components/topTable/topTable.tsx:16]: Format code here.

File: src/components/topTable/topTable.tsx [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+
 	return (
 		<div className="topTable">
-			<h1>Hello, Top Table</h1>
-			<Button variant="contained">Button-1</Button>
+			<h1>{t("table.table1Text")}</h1>
+			<Button variant="contained">{t("buttonPattern", {number: 1})}</Button>
 			<Button variant="contained" disabled>
-				Button-2
+			{t("buttonPattern", {number: 2})}
```
Comment (ExFroggy) [inline src/components/topTable/topTable.tsx:13]: Format code here.

File: src/i18n.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import i18n from 'i18next'
+import Backend from 'i18next-http-backend'
+import LanguageDetector from 'i18next-browser-languagedetector'
+import {initReactI18next} from 'react-i18next'
+
+i18n
+	.use(Backend)
+	.use(LanguageDetector)
+	.use (initReactI18next)
```
Comment (ExFroggy) [inline src/i18n.js:9]: extra space between `use` and `bracket`

File: src/components/topTable/topTable.tsx [hunk 4]
```diff
@@ -1,18 +1,22 @@
+
+import { useTranslation } from "react-i18next";
 import { Button } from "../button/button";
 
-export const topTable = () => {
+export const TopTable = () => {
+	const { t, i18n } = useTranslation();
```
Comment (ExFroggy) [inline src/components/topTable/topTable.tsx:6]: Unneccessary import of i18n.

File: src/components/bottomTable/bottomTable.tsx [hunk 5]
```diff
@@ -1,17 +1,21 @@
+import { useTranslation } from "react-i18next";
 import { Button } from "../button/button";
 
-export const bottomTable = () => {
+export const BottomTable = () => {
+	const { t, i18n } = useTranslation();
```
Comment (ExFroggy) [inline src/components/bottomTable/bottomTable.tsx:5]: Unneccessary import of i18n.

**Review by ExFroggy — COMMENTED** [review]
Please, format your code.

File: src/components/bottomTable/bottomTable.tsx (hunk 5 above)
Comment (euginios) [reply src/components/bottomTable/bottomTable.tsx]: fixed

File: src/components/topTable/topTable.tsx (hunk 4 above)
Comment (euginios) [reply src/components/topTable/topTable.tsx]: fixed

File: src/i18n.js (hunk 3 above)
Comment (euginios) [reply src/i18n.js]: fixed

File: src/components/topTable/topTable.tsx (hunk 2 above)
Comment (euginios) [reply src/components/topTable/topTable.tsx]: fixed

File: src/components/topTable/topTable.tsx (hunk 1 above)
Comment (euginios) [reply src/components/topTable/topTable.tsx]: fixed

File: src/components/header/header.tsx [hunk 6]
```diff
@@ -1,20 +1,33 @@
...
+
 	return (
 		<div className="header">
-			<h1>Shiny Palm Tree</h1>
-			<Button variant="text">Button-1</Button>
+			<h1>{t("appHeader")}</h1>
+			<Button variant="text">{t("buttonPattern", {number: 1})}</Button>
 			<Button variant="text" disabled>
-				Button-2
+			{t("buttonPattern", {number: 2})}
```
[the remark this answers is not in the document]
Comment (euginios) [reply src/components/header/header.tsx]: fixed

**Review by RPE8 — APPROVED** [review]

**Review by ExFroggy — APPROVED** [review]

**Pull request merged**